convert2rhel/data/9/x86_64/configs/oracle-9-x86_64.cfg

```diff
@@ -0,0 +1,41 @@
+[system_info]
+
+# Fingerprints of GPG keys used for signing packages of the OS to be converted.
+# The GPG key is available at http://public-yum.oracle.com/RPM-GPG-KEY-oracle-ol9
+# Delimited by whitespace(s).
+gpg_fingerprints = bc4d06a08d8b756f a7dd07088b4efbe6
```

Comment:
Can we put them in their own line? 

```suggestion
gpg_fingerprints = 
  bc4d06a08d8b756f 
  a7dd07088b4efbe6
```

Reply:
Sure!

Reply:
I don't see the changes here, @pr-watson.

Reply:
Weird, I did commit suggestion, let me manually do it